pkg/ebpf/tracer.go

```diff
@@ -85,6 +86,7 @@ type FlowFetcherConfig struct {
 	FilterConfig     *FilterConfig
 }
 
+// nolint:cyclop
```

Comment:
Can't we split the function here ?

Reply:
we probably can will do that in a follow on pr